feature-group-definitions/array-at.yml

```diff
@@ -1,5 +1,5 @@
 name: Array at()
-description: Array at() and typed array at() methods
+description: The `at()` method for arrays and typed arrays TODO.
```

Comment:
I'm wondering if we should instead say "Also supported for typed arrays" at the end of each description to keep the descriptions simpler?